### DISCUSSION TITLE
15minute market

### DISCUSSION THREAD

**Discussion by pbubik** [electricitymaps/electricitymaps-contrib#5842] [Q&A] [answered]
Europe will be transitioning to 15minute market (as opposed to 1hr market).
In the Czech Republic, this is expected to take place in May 2024.

Questions: 
1. How will Electricity Maps handle this transition? Will we average the prices and continue to display in 1hr steps? Or start displaying differents steps per country?
2. Are there any markets in the world currently that operate on other than hourly step (on wholesale side)? Are there any 15 minute markets already in operation? Where?

Thanks.

**Comment by VIKTORVAV99** [comment, marked as answer]
Hi!

This will not impact the functionality of the app or API in any way and things will continue as they are now with no change due to this. We are already parsing 15 minute data for several European countries. Price data will be a weighted average of the 4 datapoints that constitute the hour.

We have data sources that provide 5 minute resolution for example and there are some regions like Australia that has higher resolution, in Australias case I think it is 30 minutes for all of Australia and down to 5 minutes in some areas.
